### ISSUE TITLE
MESH/CL/PROX/BI-02-C, how to pass this case?

### ISSUE THREAD

**Issue by handomzhu** [intel/auto-pts#432]
it always hint to send composition data request from IUT.
but I try to send proxy connect from IUT after provision is ok and disconnect with pts, it can't create connection to pts at all. so I can't send composition data request. 

any guys can help me? thanks.

**Comment by michal-narajowski** [comment]
@handomzhu I don't think we support Mesh Proxy Client in any project right now. Are you working with some fork of auto-pts? What platform are you using?
